### ISSUE TITLE
[@types/node] stream.finished method accepts incomplete set of properties to options object

### ISSUE THREAD

**Issue by hanseartic** [DefinitelyTyped/DefinitelyTyped#70621]
### Discussed in https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/70164

<div type='discussions-op-text'>

<sup>Originally posted by **hanseartic** July 29, 2024</sup>
`FinishedOptions` accepted by `stream.finished(stream, options, callback)` are missing properties from [documentation](https://nodejs.org/docs/latest-v20.x/api/stream.html#streamfinishedstream-options-callback).

```typescript
interface FinishedOptions extends Abortable {
    error?: boolean | undefined;
    readable?: boolean | undefined;
    writable?: boolean | undefined;
}
```

`signal` and `cleanup` options

> * `signal` [\<AbortSignal\>](https://nodejs.org/docs/latest-v20.x/api/globals.html#class-abortsignal) allows aborting the wait for the stream finish. The underlying stream will not be aborted if the signal is aborted. The callback will get called with an AbortError. All registered listeners added by this function will also be removed.
> * `cleanup` [\<boolean\>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type) remove all registered stream listeners. Default: false.

are missing</div>

**Comment by hanseartic** [comment]
Checked with latest release of `@types/node`. Still missing

**Comment by Renegade334** [comment]
For reference, `cleanup` only actually exists on `stream.promises.finished()` and is currently misdocumented in Node. `signal` is indeed missing from both.

**Comment by hanseartic** [comment]
fixed per #70918
